Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -153,6 +171,38 @@ internal sealed class PythonLanguageClient : ILanguageClient, ILanguageClientCus
             _rpc.CancellationStrategy = new CustomCancellationStrategy(_server.CancellationFolderName, _rpc);
             _rpc.AllowModificationWhileListening = false;
 
+            // We also need to switch the order on handlers for all of the rpc targets. Until
+            // the VSSDK gives us a way to do this, use reflection.
+            try {
```

Comment:
This is where the internal ```targetRequestMethodToClrMethodMap``` that the json rpc object has is reversed. I had to use reflection to accomplish this.

Basically if you look here:https://github.com/microsoft/vs-streamjsonrpc/blob/640dca7d211154f93ee9771d43af2c5452a00587/src/StreamJsonRpc/Reflection/TargetMethod.cs#L44

You can see how when attempting to find a target method it finds the first item in the candidate list. 

Since the CustomMessageTarget is always the second item in the list (if it registers for the same methods as the VSSDK), then I had to reverse the list order.